### ISSUE TITLE
[UX] NT Customize Message toggle shows default message options on save but not when reloaded

### ISSUE THREAD

**Issue by unlikelyzero** [ansible/awx#4530]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
The NT Customize Message toggle shows default messages on save but not when reloaded. This can appear as though the default messages did not take.

Possible solutions
1. Make it clearer that the toggle will display default message options and if no changes are made to the messages.
2. Opening the customize message form with the toggle should keep the `isDirty` flag off until changes are made to the forms.
3. ...

##### STEPS TO REPRODUCE
1. Create Slack Notification type without customized messages
2. On Notification Templates view, click customized messages button to edit Notification created in step 1.
3. Add Customized messages without modifying codemirror fields.
4. Save and reload. Note that customized toggle is disabled.

##### EXPECTED RESULTS

1. If no changes are made to default customized messages, NT should give visual feedback to the user that nothing has changed. 

##### ACTUAL RESULTS

1. Notification Template appears to be modified and changes saved.

##### ADDITIONAL INFORMATION

![conversion](https://user-images.githubusercontent.com/12446869/63365847-6e781000-c346-11e9-9efe-69bd1e28e14e.gif)

**Comment by jladdjr** [comment]
I think https://github.com/jladdjr/awx/commit/b40566f513d604a2a2588b6ba843e26a7314a95e may fix this. (It fixed an issue I was seeing, but not sure it's the same as the one you're running into). Can you see if you can repro this again?

**Comment by keithjgrant** [comment]
Spoke with @trahman73 about this. We're OK with the behavior described here. If you don't customize the message, they toggle is effectively un-checked because the messages are still the default messages.

The 500 server error in the screen capture above does need fixing, which @jladdjr addressed above.

**Comment by unlikelyzero** [comment]
@keithjgrant updating description and title to reflect the issue

**Comment by ryanpetrello** [comment]
@unlikelyzero @jladdjr @keithjgrant what's the state of this?  Is it `state:needs_test` again, or actively being worked on by somebody?

**Comment by jladdjr** [comment]
TLDR - believe everything has been resolved, motion to close

---

@ryanpetrello - I confirmed that the 500 error has been resolved. 

Which I think just leaves the UX question about 1) toggling on 'customize messages' 2) .. but leaving the default messages in place, and 3) returning to the NT detail page and finding the toggle is been off.

Think that concern was addressed by @keithjgrant's comment here:
> Spoke with @trahman73 about this. We're OK with the behavior described here. If you don't customize the message, they toggle is effectively un-checked because the messages are still the default messages.

So I think we can close unless @unlikelyzero has any concerns about ^.

**Comment by unlikelyzero** [comment]
yea i think everything is addressed and we can close it or send it out to pasture

**Comment by jladdjr** [comment]
🐮
